lib/geocode.js

```diff
@@ -53,11 +52,30 @@ function _handleArcGISResponse (error, response, body, callback) {
  * geoservice.geocode({ text: "920 SW 3rd Ave, Portland, OR 97204" }, callback);
 */
 function geocode (parameters, callback) {
-  parameters.f = parameters.f || "json";
```

Comment:
wow. good catch.  

its actually just an [existing inconsistency](https://github.com/Esri/geoservices-js/blob/c1f5c51dc669c6ec232e750eb36f10871b0c93dd/lib/geocode.js#L88-L90) that i copied over.

i'm 8 days foggy, but willing to give myself the benefit of the doubt and say i was 'going for clarity'